.gitlab-ci.yml

```diff
@@ -162,15 +169,59 @@ default-old-autofs:
     - bundle exec rake spec_clean
     - bundle exec rake beaker:suites[default]
 
+default-old-autofs-puppet5:
```

Comment:
If there's a `default_old_autofs`, I would expect to see tests that do something with new autofs.